src/catalog/database_catalog.cpp

```diff
@@ -760,7 +760,7 @@ bool DatabaseCatalog::DeleteTable(const common::ManagedPointer<transaction::Tran
                                   const table_oid_t table) {
   if (!TryLock(txn)) return false;
   // We should respect foreign key relations and attempt to delete the table's columns first
-  auto result = DeleteColumns<Schema::Column, table_oid_t>(txn, table);
+  auto result UNUSED_ATTRIBUTE = DeleteColumns<Schema::Column, table_oid_t>(txn, table);
```

Comment:
why do you need `UNUSED_ATTRIBUTE` here? Isn't `result` being used in the next line?